packages/@aws-cdk/aws-dynamodb-global/README.md

```diff
@@ -0,0 +1,31 @@
+## @aws-cdk/aws-dynamodb-global
+
+Here is a minimal deployable Global DynamoDB tables definition:
+
+```javascript
+import globaldynamodb = require('@aws-cdk/aws-dynamodb-global');
+const CONSTRUCT_NAME = 'aws-cdk-dynamodb-global';
+const TABLE_PARTITION_KEY: Attribute = { name: 'hashKey', type: AttributeType.String };
+const STACK_PROPS: DynamoDBGlobalStackProps = {
+    dynamoProps: {
+        partitionKey: TABLE_PARTITION_KEY,
+        tableName: TABLE_NAME
+    },
+    regions: [ 'us-east-1', 'us-east-2', 'us-west-2' ]
+};
+
+const stack = new Stack();
+const globalTables = new GlobalTable(stack, CONSTRUCT_NAME, STACK_PROPS);
+```
+
+## Implementation Notes
+Global Dynamodb tables is an odd case currently.  The way this package works -
```

Comment:
Is there documentation on the recommended practice from AWS?

Reply:
AWS has no recommended practice for this since its not supported in CFN currently.  This is the solution determined by myself and by chatting with CDK contributors